esterni/manuale-sviluppatore/components/tecnologie-interessate.tex

```diff
@@ -0,0 +1,59 @@
+\documentclass[../manuale-sviluppatore.tex]{subfiles}
+
+\begin{document}
+\subsection{Back End}%
+\label{sub:back-end}
+In questa sezione sono descritte le tecnologie scelte dopo una fase di analisi per lo sviluppo del back-end.
+
+\subsubsection{InfluxDB}%
+\label{sub:influxDB}
+Abbiamo deciso di utilizzare InfluxDB, un Time Series Database open source, viste le sue alte performance e le sue elevate capacità di scrittura e di sostenere carichi di interrogazioni 
+importanti. Usiamo questo database per salvare dati momentanei, non persistenti.
+% TODO estendere
+
+\subsubsection{MySql}%
+\label{sub:mysql}
+Abbiamo deciso di utilizzare MySql per il salvataggio di dati persistenti attraverso il modello relazionale, il quale garantisce affidabilità.
```

Comment:
> relazionale, il quale garantisce

relazionale _che_ garantisce

Reply:
"il quale garantisce affidabilità" è una relativa incidentale, quindi la virgola ci sta. Fra "che" e "il quale" (in questo caso) non c'è differenza.